bin/copy-labels.js

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const { LabelHandler } = require('../lib/labels.js');
+
+const toolName = path.basename(process.argv[1]);
+const toolVersion = require('../package.json').version;
+
+const help = `\
+This tool copies labels from source repository or json file to destination
+repository or json file.
+
+Usage: ${toolName} [OPTION]... SOURCE DEST
+       ${toolName} --help
+       ${toolName} --version
+
+Options:
+  --config   Path to config file
+  --user     Github username
+  --token    Github access token
+  --delete   Delete all labels in destination repository before
+             copy
+  --update   Update existing labels with the same name
```

Comment:
The lines explaining the options should be decapitalized, to be consistent:
```suggestion
  --update   update existing labels with the same name
```